### PR TITLE
add longest increasing subsequence in JavaScript

### PR DESCRIPTION
**Pull request by anirudh-kac**
Fixes #2253 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Languages Used:
- JavaScript

#### Files Added:
- LIS.js

### CODE REVIEW

File: Longest_Increasing_Subsequence/LIS.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+Function to find the length of longest incresing subsequence
+input : an array of numbers
+returns :length of longest increasing subsequence
+*/
+
+function lengthOfLIS(nums){
+
+  //temporary array to save subsequence length values from that index
+  var arr = Array(nums.length);
```
Comment (MastersAbh) [inline Longest_Increasing_Subsequence/LIS.js:10]: looks like 2 space indentation, please change to 4 spaces

File: Longest_Increasing_Subsequence/LIS.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  //maxLen will store the result
+  maxLen = 0;
+  for(var i = 0 ; i< nums.length;i++){
```
Comment (MastersAbh) [inline Longest_Increasing_Subsequence/LIS.js:13]: space after for

File: Longest_Increasing_Subsequence/LIS.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    //update maxLen if the value at the index i is greater than the current maxLen value
+    maxLen = Math.max(maxLen,arr[i]);
+  }
+
+  return maxLen;
+}
+
+  //Examples
+
+a1  = [1,2,3];
```
Comment (MastersAbh) [inline Longest_Increasing_Subsequence/LIS.js:32]: please use dynamic inputs

**Comment by somya-kapoor** [comment]
Are you working on  it?

**Comment by MastersAbh** [comment]
Hi @anirudh-kac , I dont see any update from you for over 15 days, so we are closing the PR. in case you want to rework on this, please reach out/ raise a new PR before deadline.

**Pull request closed without merging**